tests/virtualization/universal/prepare_guests.pm

```diff
@@ -48,6 +48,8 @@ sub run {
 
     # Disable bash monitoring, so the output of completed background jobs doesn't confuse openQA
     script_run("set +m");
+    # Fix poo#101256, connection reset by peer
+    assert_script_run 'iptables -I INPUT 1 -m conntrack --ctstate INVALID -j DROP';
```

Comment:
This line drops INVALID packets, right? To be honest, I would be very surprised to see this can solve the problem. But I am not expert on this. Since you are so confident that this can solve the problem, let's see validation jobs when the network problem reproduces again.